autoload/codeium/server.vim

```diff
@@ -123,7 +123,7 @@ function! codeium#server#Start() abort
     let bin_suffix = "windows_x64.exe"
   endif
 
-  let s:root = expand('<sfile>:h:h')
+  let s:root = expand('<script>:p:h:h:h')
```

Comment:
Wondering now, should we have moved this to the top level of the file and kept it as `<sfile>:p:h:h`?